neqo-transport/src/connection/tests/migration.rs

```diff
@@ -897,6 +898,10 @@ fn retire_prior_to_migration_failure() {
             ..
         }
     ));
+
+    // See issue #1697. We had a crash when the client had a temporary path and
+    // process_output is called.
+    mem::drop(client.process_output(now()));
```

Comment:
This is good.  I'd prefer to see this as a separate test case though.

Did you not have any luck with pushing a bad packet in on a new path?  You can use the [test frame writer](https://github.com/mozilla/neqo/blob/e76645f2094365f4228b8c3b6ed8cf8694cf8849/neqo-transport/src/connection/mod.rs#L283-L284) to make a garbage packet (a single varint should do the trick, as that will be interpreted as a frame type that we don't support).

Reply:
Thanks. Will try to write another test case.